acados/sim/sim_common.c

```diff
@@ -80,14 +80,19 @@ acados_size_t sim_in_calculate_size(void *config_, void *dims)
 
     acados_size_t size = sizeof(sim_in);
 
-    int nx, nu, nz;
+    int nx, nu, nz, np = 0, np_global = 0;
```

Comment:
Initialization of to 0 not necessary as getter will overwrite it later anyway.
Comments regarding "optional" can be removed, as also "nu" and "nz" can be 0.